### PR TITLE
Update stickyfill from upstream

### PR DESCRIPTION
**Pull request by cannuk**
## WHAT

Update Stickyfill from forked repo

## WHY

I need to change this and want the latest

### CODE REVIEW

**Comment by cannuk** [comment]
Apologies that this is kind of messy. I had to do some changes based on the merge :/

**Comment by gerad** [comment]
Instead of making commit: https://github.com/sagansystems/stickyfill/pull/3/commits/1e04b0aa96085772088ad914bfd80bb2008d7c1f

Maybe you just want to revert: https://github.com/sagansystems/stickyfill/pull/3/commits/61cdc5dde568b213c9fec126cf87023e74828e4c

File: src/stickyfill.js [hunk 1]
```diff
@@ -98,8 +97,6 @@
       updateScrollPos();
       recalcAllPos();
     }
-
-    watchArray.forEach(forceRedraw);
```
Comment (gerad) [inline src/stickyfill.js]: So this is no longer an issue?

image

**Comment by gerad** [comment]
It might make sense to sit down together and clean up these commits in git to make it a bit easier to maintain going forward.

**Comment by gerad** [comment]
Also, the formatting changes caused by prettier are going to make merges way harder in the future as well.

**Comment by gerad** [comment]
> It might make sense to sit down together and clean up these commits in git to make it a bit easier to maintain going forward.

Since I don't have a lot of time to sit down together today, here's an approach that I think does the same thing but a bit more cleanly: https://github.com/sagansystems/stickyfill/pull/4

**Pull request closed without merging**